doc/fee-bumping-reserves.md

```diff
@@ -0,0 +1,43 @@
+# Fee-Bumping Reserves Recommendations for Anchor Outputs Channels
+
+The Lightning Dev Kit offers the ability to its users to open Lightning [anchor outputs](https://bitcoinops.org/en/topics/anchor-outputs/) channels (`option_anchors_zero_fee_htlc_tx`) with its counterparties.
+
+Under the Lightning security model, commitment and second-stage HTLC transactions (BOLT3's `HTLC-timeout` and `HTLC-preimage` and the corresponding spends on counterparty commitment transaction) must be realized
+in a time-sensitive manner if there are pending HTLCs. Lack of chain confirmation before HTLC expiration timelocks can constitute a loss of funds, as the counterparty can claim the HTLC output with a [competing
+transaction](https://bitcoinops.org/en/blog/waiting-for-confirmation/#policy-as-an-interface). Failure of confirmation might be provoked by blockspace demand spikes provoked by unrelated Bitcoin on-chain transaction
+issuers. Anchor output enables to remedy to this issue by allowing a Lightning node to dynamically adjust the feerate of its commitment and second-stage HTLC transactions in an unilateral way.
+
+However, anchor output channels are requiring from the user the provision and maintenance of fee-bumping reserves to attach feerate increase to the commitment (via a Child-Pay-For-Parent) or the
+second-stage HTLC transactions (via `SIGHASH_SINGLE | SIGHASH_ANYONECANPAY`). Those fee-bumping reserves should be a collection of UTXOs, of which the state and signing capabilites should be maintained
```

Comment:
That’s a good question, what level of knowledge we expect from the user reading the doc ? In my mind it’s more an application developer building on top of the raw LDK API or someone maintaining an enterprise Lightning infrastructure or advanced hobbyist.

Reply:
I’m taking the optimistic point than an application developer will search for the sighash flags and get smarter from our documentation, I can add more docs about sighash flags ? e.g https://bitcoin.stackexchange.com/questions/37093/what-goes-in-to-the-message-of-a-transaction-signature/37095#37095

Reply:
For now I took the simplest approach of just mentioning replace-by-fee. I think in the future it can be still interesting to develop the point on sighash flags, as this malleability allow external fee-bumping e.g by a LSP or watchtower.